netci.groovy

```diff
@@ -508,9 +508,6 @@ def innerLoopNonWindowsOSs = ['Ubuntu15.10', 'Debian8.2', 'OSX', 'CentOS7.1', 'O
             // Archive the results
             Utilities.addArchival(newBuildJob, "bin/build.pack,bin/osGroup.AnyCPU.${configurationGroup}/**,bin/ref/**,bin/packages/**,msbuild.log")
 
-            // Use Server GC for Ubuntu/OSX Debug PR build & test
-            def useServerGC = ((os == 'Ubuntu15.10' || os == 'OSX') && configurationGroup == 'Release' && isPR)
```

Comment:
Do you need to change the call a little bit below this? It uses the "useServerGC" variable. Line 525